spec/spec_helper.rb

```diff
@@ -1,7 +1,26 @@
 # frozen_string_literal: true
 
-require 'bundler/setup'
-require 'activeadmin-async_exporter'
+ENV['RAILS_ENV'] ||= 'test'
+require 'dummy/config/environment.rb'
+require 'pry'
+require 'active_admin'
+require 'timecop'
+
+require 'simplecov'
+SimpleCov.start
+
+Rails.application.routes.default_url_options[:host] = 'localhost:3000'
+
+require 'tmpdir'
+ActiveStorage::Blob.service =
+  ActiveStorage::Service::DiskService.new(root: Dir.mktmpdir('active_storage_tests'))
+
+ActiveRecord::Base.establish_connection(
+  adapter: 'sqlite3',
+  database: ':memory:'
+)
+
+load 'dummy/db/schema.rb'
```

Comment:
I'm not sure this is necessary if you have a full rails app in `spec/dummy` . Maybe you just need to config that in `spec/dummy/config/database.yml` ? like here https://github.com/rootstrap/active_admin_chat/blob/master/spec/dummy/config/database.yml

Reply:
It seems I do need it, since I have the same configuration on https://github.com/rootstrap/activeadmin-async_exporter/blob/tests/worker/spec/dummy/config/database.yml but when removing this line and try to run the tests I get 
```
            Failure/Error: User.find_or_create_by!(username: username)
            
            ActiveRecord::StatementInvalid:
              Could not find table 'users'

```

Reply:
Oh, nevermind, I just needed the `load 'dummy/db/schema.rb'` line

Reply:
Oh, nevermind, I had to remove the whole thing, was just removing the last line 🤦 , done!